consensus/e2e_tests/state_sync_test.go

```diff
@@ -134,28 +132,53 @@ func TestStateSyncServerGetBlock(t *testing.T) {
 	require.NotEmpty(t, getBlockRes)
 
 	require.Equal(t, uint64(1), getBlockRes.Block.GetBlockHeader().Height)
+}
+
+func TestStateSync_ServerGetBlock_FailingTest(t *testing.T) {
```

Comment:
Can you replace `FailingTest` with `Fail{Reason}`.

We might want to have other negative tests in the future as well